update waas-restart.md (#3517)
MicrosoftDocs/windows-itpro-docs: windows/deployment/update/waas-restart.md

- **Turn off auto-restart for updates during active hours** prevents automatic restart during active hours.
- **No auto-restart with logged on users for scheduled automatic updates installations** prevents automatic restart when a user is signed in. If a user schedules the restart in the update notification, the device will restart at the time the user specifies even if a user is signed in at the time. This policy only applies when **Configure Automatic Updates** is set to option **4-Auto download and schedule the install**.

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> >In case of using Remote Desktop Protocol connections, only active RDP sessions are considered as logged on users. Devices, that do not have locally logged on users or active RDP sessions, will be restarted. 

Reviewer: ```suggestion
>When using Remote Desktop Protocol connections, only active RDP sessions are considered as logged on users. Devices, that do not have locally logged on users, or active RDP sessions, will be restarted. 
```

> >In case of using Remote Desktop Protocol connections, only active RDP sessions are considered as logged on users. Devices, that do not have locally logged on users or active RDP sessions, will be restarted. 

Reviewer 2: ```suggestion
> When using Remote Desktop Protocol connections, only active RDP sessions are considered as logged on users. Devices that do not have locally logged on users, or active RDP sessions, will be restarted. 

```

>[!NOTE]
>You can only choose one path for restart behavior.
>If you set conflicting restart policies, the actual restart behavior may not be what you expected.

> >In case of using RDP, only active RDP sessions are considered as logged on users.

Reviewer: ```suggestion
>When using RDP, only active RDP sessions are considered as logged on users.
```
